apps/admin-server/src/pages/projects/[project]/areas/create.tsx

```diff
@@ -37,8 +38,12 @@ export default function ProjectAreaCreate() {
     defaultValues: {},
   });
 
-  function onSubmit(values: z.infer<typeof formSchema>) {
-    createArea(values.name, values.polygon);
+  async function onSubmit(values: z.infer<typeof formSchema>) {
+    const area = await createArea(values.name, values.polygon);
+    if (area) {
+      toast.success('Gebied aangemaakt!');
+      router.push(`/projects/${projectId}/areas`);
+    }
   }
```

Comment:
Wat als het aanmaken mislukt? Kunnen we hier dan een foutmelding tonen als toast. "Aanmaken van een gebied is niet gelukt."